src/lib/features/feature-toggle/features-read-model-type.ts

```diff
@@ -0,0 +1,3 @@
+export interface IFeaturesReadModel {
```

Comment:
introduced new read model in feature-toggle submodule so that we don't have to depend on feature-toggle-service with all write model operations.